library/common/BUILD

```diff
@@ -15,10 +15,5 @@ envoy_cc_library(
 cc_library(
     name = "envoy_jni_interface_lib",
     srcs = ["jni_interface.cc"],
-    copts = ["-std=c++14"],
-    linkopts = [
-        "-lm",
-        "-llog",
```

Comment:
I think these are still necessary for the JNI. Without these JNI builds failed to link

Reply:
The discussion we had at the time was likely it's upstream envoy that is missing `-lm` at least, but it doesn't matter there since it's being implicitly added somewhere else in the tree

Reply:
still working for me locally - @keith can you try and repro?

Reply:
I'm using a Pixel 2 XL emulator on API 28